pages/wishes/index.vue

```diff
@@ -122,6 +122,15 @@ export default {
     },
     avatarColor(fullName) {
       return randomMC.getColor({ text: fullName })
+    },
+    wishDelete(wish){
```

Comment:
Add to there **this.deleteIsLoading** changing, when action is started and finished after Promise.